### ISSUE TITLE
Obejct view: License link, toggle button sometimes not visible

### ISSUE THREAD

**Issue by Ambrosiani** [riksantikvarieambetet/Generous-Interface-Fashion#26]
In the object view, the license link and toggle button are sometimes below the fold (depending on image proportions). Some styling tweak is needed. Two examples attached w different image proportions:
![skarmavbild 2018-05-31 kl 10 13 53](https://user-images.githubusercontent.com/2394405/40770681-b307104e-64bb-11e8-9b2d-8f33087d9c2e.png)
<img width="1392" alt="skarmavbild 2018-05-31 kl 10 14 08" src="https://user-images.githubusercontent.com/2394405/40770682-b3247c42-64bb-11e8-922c-ddab681447ba.png">

**Comment by Abbe98** [comment]
Yep, we need a max height and over all styling to know what value it will have.
